modules/post/windows/gather/credentials/mremote.rb

```diff
@@ -62,7 +63,13 @@ def get_xml(path)
 
   def parse_xml(data)
 
-    mxml= REXML::Document.new(data).root
+    a = File.readlines(data)
```

Comment:
See point above, as you already have the data from `read_file()` call, so there is no need to read the file a second time.